packages/angular/projects/clr-angular/src/data/datagrid/datagrid-row.ts

```diff
@@ -261,6 +261,7 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
 
   ngOnInit() {
     this.wrappedInjector = new HostWrapper(WrappedRow, this.vcr);
+    this.selection.lockItem(this.item, this.clrDgSelectable === false);
```

Comment:
- You are accessing `this.clrDgSelectable`, but you changed it to `this.selection`. But as I said above, it's better to keep `this.clrDgSelectable` to avoid a breaking change.

- This 2nd point is a nit. Why not `!this.clrDgSelectable` instead of `this.clrDgSelectable === false`? Because as we already know `this.selection.isLocked(this.item)` is the falsy/opposite value of this.clrDgSelectable.

Reply:
I will start from bottom to top ;) 

the `this.clrDgSelectable === false` is for readdability - It's to be clear that only false is important. 

the design of this attribute is a bit misleading from the start - the idea is to disabled from selection - but you need to send it `false` as an argument. So keeping the expression more verbose is to let people calculate the end result faster (there is a comment above it ) 

And for the first one, `the.selection` is the `Selection` service that keeps track of selected element and the one that holds the methods for locking and selecting items. It's not a breaking change.